spec/acceptance/bootstrap_rpm_spec.rb

```diff
@@ -66,7 +66,7 @@
       its(:content) { should match /full_refresh_on_yum = 1/ }
       its(:content) { should match /package_profile_on_trans = 1/ }
       its(:content) { should match /hostname = #{host_inventory['fqdn']}/ }
-      its(:content) { should match /baseurl = https:\/\/#{host_inventory['fqdn']}\/pulp\/repos/ }
+      its(:content) { should match /baseurl = https:\/\/#{host_inventory['fqdn']}\/pulp\/content/ }
```

Comment:
I always like to use `%r{}` when the regex contains slashes.
```suggestion
      its(:content) { should match %r{baseurl = https://#{host_inventory['fqdn']}/pulp/content/} }
```

(same below)